.github/workflows/noosphere_apple_build.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
         with:
           brew: protobuf cmake
       - name: 'Cargo Build'
-        run: cargo build --package noosphere --release --locked --target ${{ matrix.target }}
+        run: cargo build --package noosphere --release --locked --target ${{ matrix.target }} --features rocksdb
```

Comment:
This is the only thing that should be backed out before landing.

Reply:
Just decoupling the implementation from the "on" switch -- we could make it a part of the build matrix, or configure it in `noosphere::platform`

Reply:
IDK if it's possible to distinguish e.g., M2/iPad from M2/MacBook Air in this way. We may need to talk it over a little bit to figure out the safe path forward (or face coming up with a migration playbook for orb users).